### PR TITLE
FEAT: implement PDF biulder

### PR DESCRIPTION
**Pull request by jorgepiloto**
Now that [pyansys/actions] includes a [PDF building step], we need to implement it in this project.

[pyansys/actions]: https://github.com/pyansys/actions
[PDF building step]: https://github.com/pyansys/actions/blob/f3262811fd4ed12f79fa2fc019509e5b041eaee2/doc-build/action.yml#L84-L90

### CODE REVIEW

**Comment by jorgepiloto** [comment]
Pinging @Revathyvenugopal162 here, as she is the one who implemented the builder.

File: doc/Makefile [hunk 1]
```diff
@@ -24,3 +24,9 @@ clean:
 	rm -rf $(BUILDDIR)
 	rm -rf $(SOURCEDIR)/examples
 	find . -type d -name "_autosummary" -exec rm -rf {} +
+
+# Customized pdf fov svg format images
+pdf:
+	@$(SPHINXBUILD) -M latex "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	cd build/latex && latexmk -r latexmkrc -pdf *.tex -interaction=nonstopmode || true
+	(test -f build/latex/pyansys*.pdf && echo pdf exists) || exit 1
```
Comment (Revathyvenugopal162) [inline doc/Makefile:32]:
```suggestion
	(test -f build/latex/*.pdf && echo pdf exists) || exit 1
```
Here we are testing for a pdf with name starting with pyansys exists or not, either we can skip the test or have to be generalize.

Comment (jorgepiloto) [reply doc/Makefile:32]: I see... Let us apply the changes you propose.

**Comment by jorgepiloto** [comment]
This is working fine! Requesting a review by @Revathyvenugopal162 🚀

**Review by Revathyvenugopal162 — APPROVED** [review]

**Pull request merged**